### ISSUE TITLE
The sample size in the test set

### ISSUE THREAD

**Issue by yoyowang0109** [som-shahlab/femr#254]
Hi, I'm working on implementing the evaluation part on the MIMIC-IV dataset. And I found that in your paper, "the test set is resampled with replacement to obtain 1000 bootstrap samples." Here are my questions:

1. I have read Table S16 in the paper, which lists task details for six code-based tasks on MIMIC-IV. Is the test set determined first by Table S16, and we do sampling and evaluation on it subsequently, or every time the whole test set is sampled to match Table S16?
2.  If I understand it correctly, 1000 in the context refers to the number of samples, so the sample size might not be mentioned. What is the sample size in the test set (for each task)? 
Thank you for your great help.

**Comment by EthanSteinberg** [comment]
Hi @yoyowang0109,

The splitting for MIMIC was done the same as for the other datasets, 70% training, 15% validation, 15% test, with splits based on the patient id. The same split is used for all tasks. The test set is determined before any other processing is done. (Note that because it's a split based on the total dataset, the test set for a particular task might not contain exactly 15% of the patients. It might be slightly higher or lower due to the random sampling).

The bootstrap sampling described in the paper is only for the confidence intervals / p values, etc. 1000 refers to the number of bootstraps. Each bootstrap has the same number of patients as in the original test set. For MERATIVE, the test set has 409,820 patients, so each bootstrap has 409,820 patients. **We did not compute confidence intervals for MIMIC, so we did not bootstraps for MIMIC. All MIMIC experiments were done with the single 15% test set.**

Feel free to respond or reopen this issue if you have any further questions.

**Comment by yoyowang0109** [comment]
Thanks for your explanation. So the "# Cases" and "# Control" columns in the Table S16 present the number of instances used for fine-tuning tasks. In this case, are these instances determined before any other processing is done as well? Please let me know if I misunderstood something.

**Comment by EthanSteinberg** [comment]
Correct, those instances determined before any other processing is done as well
